plugins/org.yakindu.sct.doc.user/src/user-guide/overview.textile

```diff
@@ -3,81 +3,86 @@ p.
 
-The state diagram has four states: *NorthernPort*, *SouthernPort*, *Northwards* and *Southwards*. Each of these states produces an output which is an action of the ferry. The ferry can either stop when one of the two ports is reached (_ferry.stop_) or move from one port to another (_ferry.moveSouthwards_ and _ferry.moveNorthwards_). In Moore machines the output *only* relies on the current state. This means that the output will not change unless the state also changes. This happens when a transition is taken. In our example, we have two kinds of inputs that may trigger a state transition: The ship’s mate who can decide where to sail (e.g. _mate.sailSouthwards_) and the quay which can detect an arriving ferry (e.g. _quay.reachedSouthernBank_).
+We extend the switch example above into a light switch with different brightness levels. The light switch has two buttons, an _ON_ button and an _OFF_ button. Pressing the _ON_ button switches the light on and toggles through the different brightness levels. The output of the state machine is the brightness value. As in Moore machines only states produce outputs, we need one dedicated state per brightness level:
 
-h2(#overview_mealy). Compact state machines by George H. Mealy
+!(small)images/overview_moore.png(Light switch example as a Moore machine, modeled with YAKINDU Statechart Tools)!
 
-In comparison with the Moore machines seen above, "_Mealy_ machines":https://en.wikipedia.org/wiki/Mealy_machine produce outputs only on transitions and not in states. This often results in state diagrams with fewer states because more logic can be put on transitions. As a Mealy machine, the ferry example looks like the following image:
+p=. Light switch example as a Moore machine, modeled with YAKINDU Statechart Tools
 
-!(large)images/ferry_mealy_1.png(Ferry example as a Mealy machine)!
 
-p=. Ferry example as a Mealy machine
+*Mealy machines*
 
-States do not produce outputs anymore, but the transitions do. Whether the ferry moves northwards or southwards is now expressed by the corresponding transitions. The input event mate.sail may lead to the output _ferry.moveSouthwards_ or _ferry.moveNorthwards_ depending on the active state. The state machine offers an extended logic: the states *Northwards* and *Southwards* have two transitions that lead to different states depending on the input event: If the event _mate.sail_ is raised in *Northwards* the output is _ferry.moveSouthwards_, if _quay.reachedNorthernBank_ is raised the ferry stops. The same logic as a Moore machine would lead to additional states like the following image shows:
+In comparison with Moore machines, "_Mealy_ machines":https://en.wikipedia.org/wiki/Mealy_machine *produce outputs only on transitions* and not in states. This often results in state diagrams with fewer states because more logic can be put on transitions. 
```

Comment:
"In comparison with Moore machines" -> moore kursiv

Reply:
No, italics font is only used when a term is introduced the first time. Afterwards it's normal font style again.